karma.conf-sauce.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+// Browsers to run on Sauce Labs platforms.
+var sauceBrowsers = [
+  [ "firefox", "47", "Windows 10" ],
+  [ "firefox", "46", "Windows 10" ]
+  [ "chrome", "51", "Windows 10" ],
+  [ "chrome", "50", "Windows 10" ],
+  [ "microsoftedge", "13", "Windows 10" ],
+  [ "internet explorer", "11", "Windows 10" ],
+  [ "safari", "9", "OS X 10.11" ],
+  [ "safari", "8", "OS X 10.10" ]
+].reduce(function( object, platform ) {
+  // Convert "internet explorer" -> "ie".
+  var label = platform[0].split( " " );
+  label =
+    (label.length > 1
+      ? label.map(function( word ) { return word[0]; })
+      : label
+    ).join("") +
+    "_v" + platform[1];
+
+  object[label] = {
+    base: "SauceLabs",
+    browserName: platform[0],
+    version: platform[1],
+    platform: platform[2]
+  };
+  return object;
+}, {});
+
+module.exports = function( config ) {
+  if ( !process.env.SAUCE_USERNAME || !process.env.SAUCE_ACCESS_KEY ) {
```

Comment:
It's sauce labs so not a super risk but folks could use your sauce api token. Travis and Circle have different ways to deal with it. In Circle is a bit less-robust but the linked repo above should cover an example of how to get it going. It looks like it requires a bit of maintainer info to generate the key.